docs: update documentation (#765)
unoplatform/Uno.Themes: doc/getting-started.md

# Getting Started
The Uno.Material library is available as NuGet packages that can be added to any new or existing Uno solution.

> 1. Open an existing Uno project, or create a new Uno project using the `Multi-Platform App (Uno Platform)` template.

Reviewer: ```suggestion
1. Open an existing Uno project, or create a new Uno project using the `Uno Platform App` template.
```

2. In Solution Explorer panel, right click on your solution name and select `Manage NuGet Packages for Solution ...`. Choose [`Uno.Material`](https://www.nuget.org/packages/Uno.Material/) package for Uno UWP solution or Choose the [`Uno.Material.WinUI`](https://www.nuget.org/packages/Uno.Material.WinUI) package for Uno WinUI solution. And, select the follow projects to include it:
	- PROJECT_NAME.Wasm.csproj

> 	- PROJECT_NAME.Droid.csproj
> 	- PROJECT_NAME.iOS.csproj
> 	- PROJECT_NAME.macOS.csproj

Reviewer: ```suggestion
	- `PROJECT_NAME.Mobile.csproj` (or `PROJECT_NAME.iOS.csproj`, `PROJECT_NAME.Droid.csproj`, `PROJECT_NAME.macOS.csproj` if you have an existing project)
```

	- PROJECT_NAME.Skia.Gtk.csproj
	- PROJECT_NAME.Skia.WPF.csproj

> 	- PROJECT_NAME.UWP.csproj or PROJECT_NAME.Windows.Desktop.csproj

Reviewer: ```suggestion
	- PROJECT_NAME.Windows.csproj (or PROJECT_NAME.UWP.csproj for existing projects)
```
